src/shared/api.ts

```diff
@@ -90,7 +90,7 @@ export type AnthropicModelId = keyof typeof anthropicModels
 export const anthropicDefaultModelId: AnthropicModelId = "claude-3-7-sonnet-20250219"
 export const anthropicModels = {
 	"claude-3-7-sonnet-20250219": {
-		maxTokens: 8192,
+		maxTokens: 128_000,
```

Comment:
For now, we only have a slider on the :thinking version of the models. So I think we'd want to change to 128_000 there.